docs/workflow/building/libraries/README.md

```diff
@@ -56,12 +56,12 @@ These example commands will build a release CoreCLR (and CoreLib), debug librari
 
 For Linux:
 ```bash
-./build.sh -runtimeConfiguration Release
+./build.sh -runtimeConfiguration Release -c Release
 ```
 
 For Windows:
 ```bat
-./build.cmd -runtimeConfiguration Release
+./build.cmd -runtimeConfiguration Release -c Release
```

Comment:
Thanks, but these changes aren't correct.  Adding the `-c Release` will make everything build as release, whereas the text above cites these examples are for debug libraries and debug installer.

Reply:
Thanks. That did fix my issue when building at the time, but alright. Closing.